pnparser/templates/common.py

```diff
@@ -0,0 +1,23 @@
+# static
+ONLY_INCLUDE: str = "<onlyinclude>"
+BOX_START: str = "{{Box|Start}}\n"
+BOX_BREAK: str = "{{Box|Break}}\n"
+BOX_END: str = "{{Box|End}}\n"
+LINE_BREAK: str = "<br>\n"
+THEMATIC_BREAK: str = "<hr>\n"
+TEMPLATE_END: str = "}}\n"
+PATCH_LIST_NAVBOX: str = "{{Patch List Navbox}}\n"
+PATCH_TABS_HEADER: str = "{{PatchTabsHeader}}__NOTOC__\n"
+
+# with args
+TITLE: str = "== {} ==\n"
+SUBTITLE: str = "=== {} ===\n"
+CI: str = "* {{{{ci|{}}}}}\n"
+HYPERLINK: str = "[{0} {1}]\n"
+ANCHOR: str = "{{{{Anchor|{0}}}}}'''{{{{ii|{0}}}}}'''\n"
+PATCH_HIGHLIGHTS: str = "[[File:Patch {} Banner.jpg|740px|link=]]"
+
+# raw html
+OPEN_BORDER_DIV: str = '<div style="margin:12px;border:1px solid #BBB;padding:15px 25px;width:751px;background-color:var(--interface-background)">\n'
```

Comment:
TODO: make this a css class on the wiki lmao (you can just resolve this comment immediately)